data/sql/updates/pending_db_world/rev_1737753927638017570.sql

```diff
@@ -0,0 +1,11 @@
+-- Add pure energy immune script for Vex
+DELETE FROM `creature_template` WHERE `ScriptName` = 'npc_pure_energy' AND entry = 24745;
```

Comment:
Never delete from creature_template!
Note to devs: don't edit this file, it will delete the creature if the query is reapplied